docs/user-documentation/intro-to-claw.md

```diff
@@ -52,7 +52,7 @@ The primary motive for moving to Islandora CLAW is to stay current with the repo
 ### Drupal
 [Drupal 8](https://www.drupal.org/8) has been officially released and development has begun on Drupal 9. Official Drupal policy will see Drupal 7 become unsupported when Drupal 9 is released, putting it in the same precarious territory as Fedora 3. 
 
-Current Islandora CLAW development works with Drupal 7 as a front-end, but Islandora's CLAW's structure has been built with a pivot to Drupal 8 in mind from its very inception, and work is underway to ensure that when the Islandora community is ready to switch to Drupal 8, Islandora will be there with a solid Drupal 8/Fedora 4 platform. 
+The intial phases of Islandora CLAW development worked with Drupal 7 as a front-end, but Islandora's CLAW's structure has been built with a pivot to Drupal 8 in mind from its very inception. As of June, 2016, Islandora CLAW development has pivoted fully to Drupal 8, ensuring that when the Islandora Community is ready to make the move, there will be a version of Islandora that functions with the latest and best-supported versions of both our front-end and repositry layers by pairing Drupal 8 with Fedora 4.
```

Comment:
* "The *intial* phases" - typo
* "but Islandora's CLAW's" - too many possesives
* "and *repositry* layers" - typo